tests/HomePageSmokeTests.spec.js

```diff
@@ -0,0 +1,69 @@
+const chai = require("chai");
+const expect = chai.expect;
+const App = require("../test/pageobjects/App");
+const HomePage = require("../test/pageobjects/HomePage");
+
+describe("Verifiy page logo", () => {
```

Comment:
The scenario name should describe all the covered functionalities by the tests, and here we have mentioned only the logo verification.

Reply:
Okay